pkg/controller/lifecycle/actuator_test.go

```diff
@@ -301,6 +320,94 @@ webhooks:
 `
 }
 
+func expectedManagedSeedMutatingWebhook(caBundle []byte, namespace string) string {
```

Comment:
Can you enhance the existing `expectedSeed*Webhook` functions instead of implementing new ones? The existing funcs can accept additional boolean parameter whether object selector should be empty or not. 

It could look like 

```go
func expectedShootMutatingWebhook(caBundle []byte, namespace string, withEmptyObjectSelector bool) string {
	caBundleEncoded := b64.StdEncoding.EncodeToString(caBundle)
	objectSelector := ` {}`
	if !withEmptyObjectSelector {
		objectSelector = `
    matchExpressions:
    - key: resources.gardener.cloud/managed-by
      operator: In
      values:
      - gardener`
	}

    return `apiVersion: admissionregistration.k8s.io/v1
...
  objectSelector:` + objectSelector + `
    matchExpressions:
..
}
```